great-idea-website/.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
You don't need settings.json for this repo; You will learn about how to use `.gitignore` later on.

Reply:
I think it's there because of some plug in for VS code I had at the time. I don't use VS code anymore except as my back up editor.

Reply:
@SurrealityDD correct, it is a part of your .vscode configuration file. That entire file does not need to get pushed up to GitHub. In this case, you would want to remove `.vscode` from the cache of files which your Git project tracking, then add `.vscode` to your .gitignore file so the entire folder doesn't get to GitHub.